src/App.jsx

```diff
@@ -1,33 +1,38 @@
 import './App.scss';
-import { MoviesList } from './components/MoviesList';
+import { useState } from 'react';
+
 import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
+import { FilterField } from './components/FilterField/FilterField';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
-          </div>
-        </div>
-      </div>
+function filterMovies(movies, { query }) {
+  if (query) {
+    const queryToLoverAndTrimed = query.trim().toLowerCase();
 
-      <MoviesList movies={moviesFromServer} />
-    </div>
+    return movies.filter(
+      movie => movie.title.toLowerCase().includes(queryToLoverAndTrimed)
+        || movie.description.toLowerCase().includes(queryToLoverAndTrimed),
+    );
```

Comment:
Repeatable code can be moved to helper function
```suggestion
    return movies.filter(
      movie => movie.title.toLowerCase().includes(queryToLoverAndTrimed)
        || movie.description.toLowerCase().includes(queryToLoverAndTrimed),
    );
```